spellbook/models/opensea/solana/opensea_solana_schema.yml

```diff
@@ -67,4 +67,94 @@ models:
       - *amount_usd
       - *project_contract_address
       - *evt_type
-      - *unique_trade_id
\ No newline at end of file
+      - *unique_trade_id
+
+
+  - name: opensea_solana_transfers
+    meta:
+      blockchain: solana
+      project: opensea
+      contributors: sohwak
+    config:
+      tags: ['opensea','solana','events']
+    description: >
+        OpenSea transfers on Solana
+    columns:
+      - name: blockchain
```

Comment:
for any column which has been declared previously in the same file, you can simplify it like lines 60-70 above:
for example: - *blockchain on this column

then you can apply to others below which are reused

Reply:
I changed the referenced column like `-*blockchain`, but didn't change the column order so it seems to be mixed.